src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvoker.cs

```diff
@@ -82,34 +79,32 @@ protected override void ReleaseInstance(object instance)
         {
             _controllerFactory.ReleaseController(Context, instance);
         }
-
-        protected override async Task<IActionResult> InvokeActionAsync(ActionExecutingContext actionExecutingContext)
+        
+        protected override async Task<IActionResult> InvokeActionAsync(IDictionary<string, object> arguments)
```

Comment:
This is needed so we can call this method without allocating `ActionExecutingContext`